main.go

```diff
@@ -40,30 +44,86 @@ const (
 	gcsBrowserURLPrefix = "https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs"
 )
 
-func processJobResults(suites *junit.Suites) {
-	var passed, failed, skipped, pending int
+type JobMonitoring struct {
+	Summary string `json:"summary"`
+	Errors  string `json:"errors"`
+}
+
+func sendSlackNotification(summary, errors string) error {
+	webhook := os.Getenv("WEBHOOK")
+	if webhook == "" {
+		fmt.Println("Slack Webhook is not set, skipping notification.")
+		return nil
+	}
+
+	if len(errors) > 4000 {
+		errors = errors[:4000]
+	}
+	message := JobMonitoring{
+		Summary: "Report: \n" + summary,
+		Errors:  "Errors: \n" + errors,
+	}
+
+	jsonDataMessage, err := json.Marshal(message)
+	if err != nil {
+		return fmt.Errorf("Error marshalling summary to JSON: %v\n", err)
+	}
+
+	req, err := http.NewRequest("POST", webhook, bytes.NewBuffer(jsonDataMessage))
+	if err != nil {
+		return fmt.Errorf("Error creating request: %v\n", err)
+	}
+
+	req.Header.Set("Content-Type", "application/json; charset=utf-8")
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return fmt.Errorf("Error making request: %v\n", err)
+	}
+	defer resp.Body.Close()
+
+	fmt.Printf("Job Monitoring Response status: %s\n", resp.Status)
+
+	return nil
+}
 
+func processJobResults(suites *junit.Suites, logCh chan<- string) string {
+	var passed, failed, skipped, pending int
+	var suitesSummary strings.Builder
 	for _, suite := range suites.Suites {
 		for _, result := range suite.Results {
+			// For suites where the status is empty. Ex: junit_operator.xml
+			if result.Status == "" {
```

Comment:
You should be able to add an empty string case (`case "":`) to the switch statement below to include this and drop the continue